scripts/common/_configuration.ps1

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+function loadTestProjectList()
+{
+    return Import-CSV "$RepoRoot\scripts\configuration\testProjects.csv" -Header "ProjectName"
```

Comment:
@livarcocc is the issue that we're using a data file? The file serialization format? or just that we have two sets of scripts?